tests/UI/campaigns/functional/BO/15_header/07_myProfile.js

```diff
@@ -157,7 +157,7 @@ describe('BO - My profile', async () => {
       await myProfilePage.updateEditEmployee(page, employeeData.password, employeeData);
 
       const textResult = await myProfilePage.getAlertError(page);
-      await expect(textResult).to.equal(myProfilePage.errorInvalidFormatImageMessage);
+      await expect(textResult).to.contains(myProfilePage.errorInvalidFormatImageMessage);
```

Comment:
Why changing this message ?

Reply:
Because in nightly it's not all diaplayed
See screenshot => 
![Capture d’écran de 2022-10-04 16-14-03](https://user-images.githubusercontent.com/24647495/194003491-21f11de6-e9fa-4d7f-9247-2469230aebf9.png)

Reply:
Oh sorry. I haven't got the same message.